Assets/HoloToolkit-Examples/Input/Scripts/FocusedObjectTest.cs

```diff
@@ -75,11 +78,13 @@ public void OnMakeSmaller()
 
         public void OnFocusEnter(PointerSpecificEventData eventData)
         {
+            //change the material to red color
```

Comment:
unless someone changed it to be blue. This kind of speculative assumption of what the `FocusedColor` may be shouldn't be documented.

Reply:
I just followed what I understand from the part of coding and my purpose is to provide convenience for the readers so that they can understand this easily, sorry for putting those redundant comments.